bookserver/crud.py

```diff
@@ -220,6 +220,20 @@ async def create_code_entry(data: CodeValidator):
     return CodeValidator.from_orm(new_code)
 
 
+async def fetch_code(sid: str, acid: str, course_id: int):
```

Comment:
I agree.  Also see my other PR for how we can use aliases in our pydantic models to alleviate/document some of this confusion.

Reply:
Neat, I like it! But I wonder if it would be better to rename the JS params instead?

Reply:
It could be...  I like the aliases when they solve the problem of allowing you to use camelCase in your JS code and snake_case in your python.

again, my main frame of mind is to port these endpoints as quickly as I can with as little disruption to other code as I can.  Maybe that is not the right attitude but it is how I've been thinking about it.

Reply:
I see the approach, and I understand it. My suggestion: start providing links from an endpoint to the place in the JS where the endpoint is called, so it's at least easier to see what both sides are doing.

Reply:
OK, good suggestion.  I've done that and will try to do that consistently.